analyzers/its/regression-test.ps1

```diff
@@ -478,28 +470,31 @@ try {
 
     Write-Header "Processing analyzer results"
 
-    CheckInternalProjectsDifferences
-
-    Write-Host "Normalizing the SARIF reports"
-    $sarifTimer = [system.diagnostics.stopwatch]::StartNew()
-
-    # Normalize & overwrite all *.json SARIF files found under the "actual" folder
-    Get-ChildItem output -filter *.json -recurse | where { $_.FullName -notmatch 'ManuallyAddedNoncompliantIssues' } | Foreach-Object { New-IssueReports $_.FullName }
-
-    $sarifTimerElapsed = $sarifTimer.Elapsed.TotalSeconds
-    Write-Debug "Normalized the SARIF reports in '${sarifTimerElapsed}'"
-
     Write-Host "Computing analyzer performance"
     $measurePerfTimer = [system.diagnostics.stopwatch]::StartNew()
     Measure-AnalyzerPerformance
     $measurePerfTimerElapsed = $measurePerfTimer.Elapsed.TotalSeconds
     Write-Debug "Computed analyzer performance in '${measurePerfTimerElapsed}'"
 
-    Write-Host "Checking for differences..."
-    $diffTimer = [system.diagnostics.stopwatch]::StartNew()
-    Show-DiffResults
-    $diffTimerElapsed = $diffTimer.Elapsed.TotalSeconds
-    Write-Debug "Checked for differences in '${diffTimerElapsed}'"
+    CheckInternalProjectsDifferences
+
+    # Not needed when $project is internal
+    if ($project -eq "" -or -not $InternalProjects.Contains($project)) {
```

Comment:
The `expected` directory doesn't exist for internal projects. They are compared in `CheckInternalProjectsDifferences`. Further steps were useless, noisy and failing.